elastic/kibana
# [Portable Dashboards] By Value ID

## Summary
This PR fixes https://github.com/elastic/kibana/issues/165362 by adding a `by value id` which can be used as a backup in place of the Saved Object ID for any systems that require one. This includes session storage backups and determining if the Dashboard is saved or not when the Dashboard is created by value.

How to test:
1. Navigate to Observability (You will need some appropriate data to test - I used OBLT) -> services
2. Open the `opbeans-java` service. 
3. Click on the `metrics` tab. You should see a Dashboard load up.
4. On any panel, click the `...` icon, and click `copy to Dashboard`.
5. You should be able to copy the panel to a `new` dashboard.

Before this PR, the `new` button was disabled because the Dashboard could not distinguish between a Dashboard loaded by value, and a newly created Dashboard. This made the copy to Dashboard panel believe that you were already editing a `new` Dashboard.
